java/runtime/src/main/java/org/ray/runtime/AbstractRayRuntime.java

```diff
@@ -125,31 +73,7 @@ protected void resetLibraryPath() {
 
   @Override
   public <T> RayObject<T> put(T obj) {
-    ObjectId objectId = ObjectId.forPut(workerContext.getCurrentTaskId(),
-        workerContext.nextPutIndex());
-    put(objectId, obj);
-    return new RayObjectImpl<>(objectId);
-  }
-
-  public <T> void put(ObjectId objectId, T obj) {
-    TaskId taskId = workerContext.getCurrentTaskId();
-    LOGGER.debug("Putting object {}, for task {} ", objectId, taskId);
-    objectStoreProxy.put(objectId, obj);
-  }
-
-
-  /**
-   * Store a serialized object in the object store.
-   *
-   * @param obj The serialized Java object to be stored.
-   * @return A RayObject instance that represents the in-store object.
-   */
-  public RayObject<Object> putSerialized(byte[] obj) {
-    ObjectId objectId = ObjectId.forPut(workerContext.getCurrentTaskId(),
-        workerContext.nextPutIndex());
-    TaskId taskId = workerContext.getCurrentTaskId();
-    LOGGER.debug("Putting serialized object {}, for task {} ", objectId, taskId);
-    objectStoreProxy.putSerialized(objectId, obj);
+    ObjectId objectId = objectStoreProxy.put(obj);
```

Comment:
I think we can get rid of the `ObjectStoreProxy` class now, and just use `objectStore.put` here.